tests/src/system/basic/CLIRuleTests.java

```diff
@@ -65,9 +66,9 @@ public static void setUp() throws Exception {
     @Test
     public void rule1to1() throws Exception {
         try {
-        	wsk.sanitize(Rule, "R_121");
-        	wsk.sanitize(Action, "A_121");
-        	wsk.sanitize(Trigger, "T_121");
+            wsk.sanitize(Rule, "R_121");
```

Comment:
surely if there are tabs now, either we're not running scan code correctly, or there are no tabs.